src/ripple/app/ledger/Consensus.h

```diff
@@ -74,6 +74,11 @@ class Consensus
     void
     setLastCloseTime (std::uint32_t t) = 0;
 
+    /** Get the network time when the last ledger closed */
+    virtual
+    std::uint32_t
+    getLastCloseTime () const = 0;
```

Comment:
True but if you aren't going to make the change in this pull request you should at least make the change mentally.